package.json

```diff
@@ -15,6 +15,7 @@
   "main": "packages-ts/starknet-gauntlet-cli/dist/index.js",
   "bin": "packages-ts/starknet-gauntlet-cli/dist/index.js",
   "scripts": {
+    "pregauntlet": "command -v tsc >/dev/null 2>&1 || yarn install",
```

Comment:
This checks if `tsc` is present and does yarn install if it is not. Needed for the remote runner

Reply:
Where is this `"pregauntlet"` script defined? How do we know it needs to be added here?

Not a fan of this type of integration, very implicit and brittle. 
We'd need to explicitly define what a remote runner is, why is it required, how do we set it up.

Reply:
This is a npm check basically, if we prepend `pre` to any existing script it will execute prior to the script which in our case is `gauntlet`, so each time we invoke `gauntlet` the `pregauntlet` will run. This checks if tsc is installed since by default we don't install, we only build so it causes some packages to be missing, if it is not installed it performs `yarn install` which installs all the deps